### PR TITLE
All day event

### PR DESCRIPTION
**Pull request by Eyalevin**
- add all day event (fix issue  #15)

### CODE REVIEW

**Comment by Eyalevin** [comment]
@freekmurze I add an example to the readme look in the main example in the readme

File: src/Link.php [hunk 1]
```diff
@@ -29,31 +29,51 @@ class Link
...
             throw InvalidLink::invalidDateRange($from, $to);
         }
 
-        $this->from = $from;
-        $this->to = $to;
+        if ($this->allDay) {
+            $this->from = $from->format('Ymd');
+            $this->to = $to->format('Ymd');
+
+        } else {
```
[the remark this answers is not in the document]
Comment (Eyalevin) [reply src/Link.php]: After passing this code again I agree with you

File: src/Link.php [hunk 2]
```diff
@@ -29,31 +29,51 @@ class Link
...
 
+    /** @var boolean */
+    protected $allDay;
+
     /** @var string */
     protected $address;
 
-    public function __construct(string $title, DateTime $from, DateTime $to)
+    /**
+     * Link constructor.
```
[the remark this answers is not in the document]
Comment (Eyalevin) [reply src/Link.php]: I think this is important and helps to read the code

[the remark this answers is not in the document]
Comment (freekmurze) [reply src/Link.php]: Remove it still so follows how code blocks are used in all our other packages.

**Comment by Eyalevin** [comment]
@freekmurze I add  commit

**Comment by mbardelmeijer** [comment]
Would it be possible to get this merged? Happy to help if there are any changes required.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**